cookbook/specs/fvm.json

```diff
@@ -1831,21 +2833,165 @@
                             },
                             {
                               "name": "pruning",
+                              "parse_directive": {
+                                "function_tag": "GET_BLOCK_BY_NUM"
+                                },
                               "values": [
                                 {
-                                  "expected_value": ""
+                                  "latest_distance": 2800
                                 },
                                 {
                                   "extension": "archive",
-                                  "expected_value": ""
+                                  "expected_value": "1"
```

Comment:
according to the filecoin docs: https://docs.filecoin.io/networks/mainnet, it began on block 148888, not 1